src/shacl-shape-lib/connectors-shacl-shape.xsl

```diff
@@ -360,40 +473,52 @@
                     ()"/>
         <xsl:variable name="targetClassURI"
             select="f:buildURIfromLexicalQName($connector/target/model/@name)"/>
+        <xsl:variable name="normalisedTargetClassName"
+            select="replace($connector/target/model/@name, ':', '-')"/>
```

Comment:
@Dragos0000 a professional would have never copy-pasted the SAME code so many times. 

Let's be professionals. Follow the principle: Write once, reuse many times.